modules/virt-editing-vm-web.adoc

```diff
@@ -41,6 +41,7 @@ Labels and annotations are editable for both preconfigured Red Hat templates and
 . Click the *{object-gui}* tab.
 
 . Select a {object}.
+.. Optional: Use the *Filter* drop-down menu to filter the list of virtual machines by attributes such as status, template, node, or operating system (OS).
```

Comment:
- Because this applies to both templates and VMs, I would either remove "of virtual machines" or change it to `{object}s`. 
- Maybe s/to filter the list/to sort the list (to avoid too much repetition)
- I wonder if this should be its own (optional) step *before* `Select a {object}`, instead of a sub-step of `. Select a {object}`